### PR TITLE
Finalize README

### LINKED ISSUE #1: Further improve README
README is at the moment not enough documented.

### PR DESCRIPTION
**Pull request by damianooldoni**
In this PR I did the following:
1. fix #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # git-course-demo
 Test wathever I want today with my colleagues.
+
+This git course is inspired and partly given using a [pdf](https://inbo.github.io/git-course/static/presentations/git.pdf).
```
Comment (PietrH) [inline README.md:4]:
```suggestion
This git course is inspired and partly given using a [pdf](https://inbo.github.io/git-course/static/presentations/git.pdf), originally written by Peter Desmet and Stijn Van Hoey. 
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # git-course-demo
 Test wathever I want today with my colleagues.
+
+This git course is inspired and partly given using a [pdf](https://inbo.github.io/git-course/static/presentations/git.pdf). 
+
+Something extra will be added next year.
```
Comment (PietrH) [inline README.md:6]:
```suggestion
Something extra will be added next year (2024).
```

**Comment by RCinbo** [comment]
Very nice work, Damiano!

**Review by RCinbo — APPROVED** [review]

**Comment by damianooldoni** [comment]
tHANKS @PietrH AND @RCinbo; i IWLL MERGE NOW;

**Pull request merged**